### DISCUSSION TITLE
Can Graphics Card (Intel Iris Xe) used for CARLA ?

### DISCUSSION THREAD

**Discussion by shauryaneel** [carla-simulator/carla#5576] [Q&A]
Dear members,

Can I run CARLA simulator with Intel(R) Iris Xe graphics card ?

**Comment by lorinachey** [comment]
I know this post was over a year ago, but I was unable to get CARLA to run with an Intel Iris Xe Graphics card on an ASUS ZenBook. I tried lowering the resolution and still could not run it. This is on Windows 10.

**Reply by 3F999** [reply]
you can run it, with this system just klick on the address bar in your carla folder then from that path open cmd just write cmd then write:  CarlaUE4.exe -dx11 
and your carla will open

**Reply by lorinachey** [reply]
This gets me further along - CARLA will actually open whereas before it would crash with an out of memory error, but it's still basically frozen on my machine. Maybe it's just my laptop but it seems to be really struggling.

**Reply by 3F999** [reply]
ya for me also but with this code that i sent you the problem solved

**Reply by 3F999** [reply]
you can check also your GPU and if some thing use your graffic card you can reduce it

**Reply by lorinachey** [reply]
After playing around with this a little more, this solution does not allow CARLA to run in any meaningful capacity. I can get CARLA to open, however my GPU is at 100%. If I try running any of the examples in the PythonAPI directory that comes with CARLA, it crashes. This is with no other programs running on my machine. Based on this experience, I would not recommend trying to run CARLA on the Iris Xe graphics card. This isn't too surprising since the docs recommend a dedicated GPU.

**Comment by 3F999** [comment]
Yes Exactly thats true about Graffic card, but still there is different ways to run it finally if you want we can talk in email is more faster than here

**Reply by Aimo2027** [reply]
I have an Intel(R) Iris(R) Xe graphics card and would like to use the CARLA software. Could you please share guidance on how I can set it up and use it effectively?

Thank you for your assistance!

**Comment by 3F999** [comment]
Dear Aimo, 

thank you for your email, In this case you need to check the capacity of your graffic card, for Carla you need to have access to minimum 6G GPU, more over about 20GB disk space, then you need to an IDE recommended Pycharm and make an virtual enviromnt inside it and install Python version 3.7 and download Carla version like 0.9.13 (depend on Carla version try to install the suitable version of python).you can follow the rest of the instructions from my github: GitHub - 3F999/CARLA: Simulation Autonomous Vehicles
I hope this instructions helps you.
Best regards,Behnaz Fayazi


    On Tuesday, November 19, 2024 at 03:49:56 AM GMT+1, Aimo2027 ***@***.***> wrote:  
 
 


I have an Intel(R) Iris(R) Xe graphics card and would like to use the CARLA software. Could you please share guidance on how I can set it up and use it effectively?

Thank you for your assistance!
